lib/api/routes/iterations.rb

```diff
@@ -48,12 +48,19 @@ class Iterations < Core
           halt 401, {error: message}.to_json
         end
 
+        #rubocop code analysis start
+         rubocop_code_file = File.new("#{settings.root}/rubocop_tmp/test_#{user.id}.rb", 'w+')
```

Comment:
Line is too long. [94/80]<br>Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.